README.md

```diff
@@ -51,6 +51,7 @@ Learn more at <https://www.amplify.com>
 3.  Repository level plan/apply. Terrawrap provides commands for running plan/apply recursively on a entire
     repository at once.
 
+4. Repository level dependency visualization. Terrawrap provides commands for displaying the order of applies in human readable output.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[maximum-line-length] Line must be at most 120 characters](https://app.codacy.com/manual/amplify-education/terrawrap/pullRequest?prid=4739316)